### PR TITLE
bots: Have curl write directly to disk in image-download

### PR DESCRIPTION
**Pull request by stefwalter**
Lets simplify the logic here, and have curl write directly
to disk. We're trying to solve some strange corner cases
failing in the tests.

### CODE REVIEW

File: bots/image-download [hunk 1]
```diff
@@ -139,14 +139,16 @@ def download(link, force, quiet, stores):
     if not quiet:
         cmd.remove("--silent")
         cmd.insert(1, "--progress-bar")
+    cmd.append("--output")
+    cmd.append(temp)
```
Comment (martinpitt) [inline bots/image-download]: I think for this you should close `fd` *before* calling the process, otherwise you have the file open for writing twice (once here and once from curl).

Comment (stefwalter) [inline bots/image-download]: Fixed.
